store/sqlstore/team_store.go

```diff
@@ -1061,31 +1134,61 @@ func (s SqlTeamStore) RemoveMember(teamId string, userId string) *model.AppError
 }
 
 func (s SqlTeamStore) RemoveAllMembersByTeam(teamId string) *model.AppError {
-	_, err := s.GetMaster().Exec("DELETE FROM TeamMembers WHERE TeamId = :TeamId", map[string]interface{}{"TeamId": teamId})
+	sql, args, err := s.getQueryBuilder().
+		Delete("").From("TeamMembers").
+		Where(sq.Eq{"TeamId": teamId}).ToSql()
+	if err != nil {
+		return model.NewAppError("SqlTeamStore.RemoveMembers", "store.sql.build_query.app_error", nil, "team_id="+teamId+", "+err.Error(), http.StatusInternalServerError)
+	}
+
+	_, err = s.GetMaster().Exec(sql, args...)
 	if err != nil {
 		return model.NewAppError("SqlTeamStore.RemoveMember", "store.sql_team.remove_member.app_error", nil, "team_id="+teamId+", "+err.Error(), http.StatusInternalServerError)
 	}
 	return nil
 }
 
 func (s SqlTeamStore) RemoveAllMembersByUser(userId string) *model.AppError {
-	_, err := s.GetMaster().Exec("DELETE FROM TeamMembers WHERE UserId = :UserId", map[string]interface{}{"UserId": userId})
+	sql, args, err := s.getQueryBuilder().
+		Delete("").From("TeamMembers").
+		Where(sq.Eq{"UserId": userId}).ToSql()
+	if err != nil {
+		return model.NewAppError("SqlTeamStore.RemoveMembers", "store.sql.build_query.app_error", nil, "team_id="+userId+", "+err.Error(), http.StatusInternalServerError)
+	}
+	_, err = s.GetMaster().Exec(sql, args...)
 	if err != nil {
 		return model.NewAppError("SqlTeamStore.RemoveMember", "store.sql_team.remove_member.app_error", nil, "user_id="+userId+", "+err.Error(), http.StatusInternalServerError)
 	}
 	return nil
 }
 
 func (s SqlTeamStore) UpdateLastTeamIconUpdate(teamId string, curTime int64) *model.AppError {
-	if _, err := s.GetMaster().Exec("UPDATE Teams SET LastTeamIconUpdate = :Time, UpdateAt = :Time WHERE Id = :teamId", map[string]interface{}{"Time": curTime, "teamId": teamId}); err != nil {
+	sql, args, err := s.getQueryBuilder().
+		Update("").Table("Teams").
```

Comment:
I think we can just do `Update("Teams")` here.

Reply:
Done.